docs/usage.md

```diff
@@ -95,6 +95,28 @@ genome: 'GRCh37'
 
 You can also generate such `YAML`/`JSON` files via [nf-core/launch](https://nf-co.re/launch).
 
+### Decontanmination
+
+Filter the output files from metagenomics classifiers like `Kraken2`, `Centrifuge`, or `DIAMOND` to remove false positives and background contamination can be activated with `--decontamination`. This step compares results to the negative control to identify likely present species based on user-defined thresholds, effecitively eliminating background noise from microbiome species.
+
+### Extract Viral TaxIDs
+
+This step extracts all taxonomic IDs of viral species predicted by classifers. Switching on by `--perform_extract_reads`
+
+### Extract Reads
+
+This step either pulls the reads of all viral TaxIDs predicted by classifiers or extracts reads from a user-defined list of TaxIDs separated by spaces when the `--taxid` option is activated. Extracting reads predicted by `Kraken2` can be activated with `--extract_kraken2_reads`, extracting reads predicted by `Centrifuge` can be activated with `--extract_centrifuge_reads` and extracting reads predicted by `DIAMOND` can be activated with `--extract_diamond_reads`.
+
+If the `--taxid` option is included in the command line, the pipeline will only extract reads for the specified TaxIDs, in other words, `--taxid` takes priority.
+
+### de-novo assembly
+
+De novo assembly can be performed for extracted reads of TaxIDs by disabling `--skip_shortread_denovo` for short reads or the `--skip_longread_denovo` option for long reads, provided the number of reads exceeds `params.min_read_counts`. The recommended minimum number of reads is 100. If there are too few reads, the process will fail.
```

Comment:
So, de-novo assembly is the default? I find it a bit confusing with the `skip` prefix in front of the denovo-related parameters.

Reply:
I feel same. Will change it into `perform_denovo`

Reply:
Done!